src/test/java/com/epam/reportportal/cucumber/LaunchLoggingContextTest.java

```diff
@@ -0,0 +1,44 @@
+package com.epam.reportportal.cucumber;
+
+import com.epam.reportportal.listeners.ListenerParameters;
+import com.epam.reportportal.service.Launch;
+import com.epam.reportportal.service.ReportPortal;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.*;
+
+/**
+ * @author <a href="mailto:ihar_kahadouski@epam.com">Ihar Kahadouski</a>
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class LaunchLoggingContextTest {
+
+	@Mock
+	private ReportPortal reportPortal;
+
+	@Mock
+	private ListenerParameters listenerParameters;
+
+	@Mock
+	private Launch launch;
+
+	@Test
+	public void verifyLaunchLoggingContextInit() {
+		StepReporter stepReporter = new StepReporter() {
```

Comment:
The same as for this review: https://github.com/reportportal/agent-java-cucumber/pull/55

Reply:
done